wordpress-mobile/WordPress-iOS
# UGC: Track post author reported event

Fixes #20133

This PR tracks the event `reader_post_author_reported` when the user taps **Report this user** button.

## Test Instructions

1. Build and run WordPress in Xcode. 
2. Log in then head to **Reader > Following** or **Reader > Discover.**
3. Tap ⠇button on any post.
4. Tap **Report this user** button.
5. **Verify** the following event is logged in Xcode Console.
```
🔵 Tracked: reader_post_author_reported <blog_id: xyz, feed_id: xyz, feed_item_id: xyz, is_jetpack: 0, post_author_id: xyz, post_id: xyz, subscription_count: xyz>
```

## Regression Notes
1. Potential unintended areas of impact
None.

6. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests.

7. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
